### ISSUE TITLE
Toolbars not renedered with borders

### ISSUE THREAD

**Issue by lpgb** [freemint/freemint#231]
My app uses toolbars which have a border. The border is set with bfobspec -> framesize set to out 1 pixel. NAES renders the border exactly as created in the resource editor but XaAES does not. The border is missing which in turn causes the window work area to be 2 scan lines larger than expected. Attached are some screen shots. I've tested with Atari's AES 4.92 and it renders the border as well.

I've done some extreme tests with borders of 'out 4 pixels' and both NAES and AES 4.92 get it right, but I didn't screen shot those. It's my opinion XaAES is a fault not mimicking what these other AES's are doing. It should not ignore bfobspec -> framesize if it's set to 'out x pixels'.

NAES
![naes](https://user-images.githubusercontent.com/25260066/135492781-c33a0f8b-3b30-4772-bef8-b2b4ba28bf81.png)

AES 4.92
![aes492](https://user-images.githubusercontent.com/25260066/135492868-e946b880-0e4c-4d2c-a47d-205374808994.png)

XaAES (toolbar missing border)
![xaaes](https://user-images.githubusercontent.com/25260066/135492926-9651f28a-0785-451a-b296-839428538840.png)

**Comment by th-otto** [comment]
Yes, that seems to be wrong. I can only imagine that this is caused by the more generic widget drawing routines in XaAES.

Do N.AES and AES 4.92 also react on other attributes, like outlined & shadowed? This would add even more pixels to the outline area of normal objects.

**Comment by th-otto** [comment]
PS.: are you writing an ZView clone there? ;)

**Comment by lpgb** [comment]
Have not tested other attributes, they wouldn't look good, but if someone is going to work on it they should be tested. No it's not a zview clone, just an image viewer using codecs with some misc features thrown in for fun. lol

**Comment by lpgb** [comment]
Tried to render the border in 1 pixel thinking that might work, but it does not. Seems to completely ignore borders.

**Comment by mikrosk** [comment]
I need some test utility. ImgView refuses to work with fVDI and I can't install NVDI in Aranym right now.
